### DISCUSSION TITLE
config_path on Error Analysis Plots & Evaluation

### DISCUSSION THREAD

**Discussion by saintrealchoi** [obss/sahi#645] [Q&A] [answered]
First of all, Thanks for your amazing framework.

on below code

```
sahi predict --source image_dir/ --dataset_json_path dataset.json --model_type mmdet --model_path weight.pt --config_path config.yaml --no_sliced_prediction
```

I think options `config_path` should be changed `model_config_path`.

Sincerely,
Sungjin

**Comment by fcakyon** [comment, marked as answer]
@saintrealchoi thanks a lot for mentioning this issue! I have updated the type in this post: https://github.com/obss/sahi/issues/356#issue-1101552018

**Comment by saintrealchoi** [comment]
@fcakyon Thx!!
